src/main/java/de/tum/cit/aet/artemis/athena/config/AthenaAuthorizationInterceptor.java

```diff
@@ -16,7 +14,7 @@
 import org.springframework.stereotype.Component;
 
 @Component
-@Profile(PROFILE_ATHENA)
+@ConditionalOnBean(AthenaConfiguration.class)
```

Comment:
Instead of relying on the profile, we rely on the existence of the `AthenaConfiguration` (which checks the environment)